Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -9327,6 +9660,86 @@ def download_software(topology: Topology, version: str,
     return UniversalCommand.download_software(topology, version, device_filter_str=device_filter_string, sync=_sync)
 
 
+def install_software(topology: Topology, version: str,
+                     device_filter_string: Optional[str] = None, sync: Optional[bool] = False) -> InstallSoftwareCommandResult:
+    """
+    Install the given software version onto the device. Download the software first with
+    pan-os-platform-download-software
+
+    :param topology: `Topology` instance !no-auto-argument
+    :param device_filter_string: String to filter to only install to specific devices or serial numbers
+    :param version: software version to upgrade to, ex. 9.1.2
+    :param sync: If provided, runs the download synchronously - make sure 'execution-timeout' is increased.
+    """
+    _sync = argToBoolean(sync)
+    return UniversalCommand.install_software(topology, version, device_filter_str=device_filter_string, sync=_sync)
+
+
+def reboot(topology: Topology, hostid: str) -> RestartSystemCommandResult:
+    """
+    Reboot the given host.
+
+    :param topology: `Topology` instance !no-auto-argument
+    :param hostid: ID of host (serial or hostname) to reboot
+    """
+    return UniversalCommand.reboot(topology, hostid=hostid)
+
+
+def system_status(topology: Topology, hostid: str) -> CheckSystemStatus:
+    """
+    Checks the status of the given device, checking whether it's up or down and the operational mode normal
+
+    :param topology: `Topology` instance !no-auto-argument
+    :param hostid: ID of host (serial or hostname) to check.
+    """
+    return UniversalCommand.check_system_availability(topology, hostid=hostid)
+
+
+def update_ha_state(topology: Topology, hostid: str, state: str) -> HighAvailabilityStateStatus:
+    """
+    Checks the status of the given device, checking whether it's up or down and the operational mode normal
+
+    :param topology: `Topology` instance !no-auto-argument
+    :param hostid: ID of host (serial or hostname) to update the state.
+    :param state: New state.
+    """
+    return FirewallCommand.change_status(topology, hostid=hostid, state=state)
+
+
+"""Hygiene Commands"""
+
+
+def check_log_forwarding(topology: Topology,
+                         device_filter_string: Optional[str] = None) -> ConfigurationHygieneCheckResult:
+    """
+    :param topology: `Topology` instance !no-auto-argument
+    :param device_filter_string: String to filter to only check given device
+    """
+    return HygieneLookups.check_log_forwarding_profiles(topology, device_filter_str=device_filter_string)
+
+
+def check_vulnerability_profiles(
+        topology: Topology,
+        device_filter_string: Optional[str] = None,
+        minimum_block_severities: str = "critical,high",
+        minimum_alert_severities: str = "medium,low"
+) -> ConfigurationHygieneCheckResult:
```

Comment:
can you state in docstring whether this function should be used for both panorama and firewall or only for one of them?